service/storage_service.cc

```diff
@@ -119,13 +119,14 @@ distributed<storage_service> _the_storage_service;
 
 timeout_config make_timeout_config(const db::config& cfg) {
     timeout_config tc;
-    tc.read_timeout = cfg.read_request_timeout_in_ms() * 1ms;
-    tc.write_timeout = cfg.write_request_timeout_in_ms() * 1ms;
-    tc.range_read_timeout = cfg.range_request_timeout_in_ms() * 1ms;
-    tc.counter_write_timeout = cfg.counter_write_request_timeout_in_ms() * 1ms;
-    tc.truncate_timeout = cfg.truncate_request_timeout_in_ms() * 1ms;
-    tc.cas_timeout = cfg.cas_contention_timeout_in_ms() * 1ms;
-    tc.other_timeout = cfg.request_timeout_in_ms() * 1ms;
+    tc.read_timeout = cfg.read_request_timeout_in_ms;
+    tc.write_timeout = cfg.write_request_timeout_in_ms;
```

Comment:
Since those values are no longer durations, their names should have _ms suffixes to indicate the units.

Even better, please see if you can make the configuration item store a duration instead of an integer. Maybe more glue logic is needed, but in the end it will save a lot of unit mistakes.